connectors/src/resources/zendesk_resources.ts

```diff
@@ -485,6 +485,22 @@ export class ZendeskTicketResource extends BaseResource<ZendeskTicket> {
     super(ZendeskTicket, blob);
   }
 
+  static async makeNew({
+    blob,
+    transaction,
+  }: {
+    blob: CreationAttributes<ZendeskTicket>;
+    transaction?: Transaction;
+  }): Promise<ZendeskTicketResource> {
+    let article;
+    if (transaction) {
```

Comment:
not sure you need the "if"

Reply:
handled in https://github.com/dust-tt/dust/pull/8314